server/routes.js

```diff
@@ -206,7 +209,7 @@ function authenticateRequestCasa (req, res, next) {
 function authenticateRequestCallback (req, res, next) {
   logger.log2('verbose', `Authenticating request against ${req.params.provider}`)
   passport.authenticate(
-    req.params.provider, { failureRedirect: '/passport/error', failureFlash: true }
+    req.params.provider, { failureRedirect: '/passport/error', failureFlash: true, session: false }
```

Comment:
Is this set to false by default in the previous version of the API?

Reply:
When comparing the two versions, it seems like the session manager implementation in v6 was updated to require a express session middleware. To keep similar behaviour as v5, you would have to explicitly set the session to false.

https://github.com/jaredhanson/passport/compare/v0.5.3...v0.6.0#diff-ba75157e9d5273ecf42d4a7d4538e5dad57b5aa0774cbd86ebaf109bb144c86cR21